src/scss/_objects.content-card.scss

```diff
@@ -0,0 +1,41 @@
+.content-card {
+  margin: halve($inuit-base-spacing-unit) 0;
+
+  > .box {
+    background-color: #fff;
+  }
+
+  &__thumbnail {
+    width: 270px;
```

Comment:
Just asking because I know this is something the core Grommet team tends to bring up - is it possible to use an existing Grommet size variable for this?  Or if not, can this be a multiple of the `$inuit-base-spacing-unit`?  If that's not an option either, then it might be good to at least use a variable for it locally, since this value is used several times.  :)